proto/connectrpc/conformance/v1alpha1/server_compat.proto

```diff
@@ -42,15 +42,8 @@ message ServerCompatRequest {
 
 // The outcome of one ServerCompatRequest.
 message ServerCompatResponse {
-  oneof result {
```

Comment:
I think we talked about this change. I went ahead and made it since it simplifies the test runner to now have to consider the error case this way.